app/ui/lib/Combobox.tsx

```diff
@@ -177,13 +177,21 @@ export const Combobox = ({
             </ComboboxButton>
           )}
         </div>
-        {items.length > 0 && (
+        {(items.length > 0 || allowArbitraryValues) && (
           <ComboboxOptions
             anchor="bottom start"
             // 14px gap is presumably because it's measured from inside the outline or something
-            className={`ox-menu pointer-events-auto ${zIndex} relative w-[calc(var(--input-width)+var(--button-width))] overflow-y-auto border !outline-none border-secondary [--anchor-gap:14px] empty:hidden`}
+            className={`ox-menu pointer-events-auto ${zIndex} relative w-[calc(var(--input-width)+var(--button-width))] overflow-y-auto border !outline-none border-secondary [--anchor-gap:13px] empty:hidden`}
             modal={false}
           >
+            {allowArbitraryValues && filteredItems.length === 0 && (
```

Comment:
Maybe it should show up even when there are matches as long as it doesn't exactly match any of the matches?

Reply:
Ah, so if `tacocat` is an existing option in the list, and they've typed in `taco`, we should show `tacocat` as well as `Use taco`. Sounds good. Will sort that out.